numpy/core/src/multiarray/compiled_base.c

```diff
@@ -691,8 +1059,8 @@ arr_interp_complex(PyObject *NPY_UNUSED(self), PyObject *args, PyObject *kwdict)
     npy_intp i, lenx, lenxp;
 
     const npy_double *dx, *dz;
-    const npy_cdouble *dy; 
-    npy_cdouble lval, rval; 
+    const npy_cdouble *dy;
+    npy_cdouble lval, rval;
```

Comment:
If you're going to touch this unrelated whitespace, can you do it in a separate commit?

Reply:
no problem. should it be in a separate PR as well?

Reply:
these two lines were missed. all the other whitespace changes are on a separate commit now as part of this PR.